### PR TITLE
init skeleton v0

### PR DESCRIPTION
**Pull request by pkrasovskyi**
Let's make true skeleton

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>2.1.7.RELEASE</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
```
Comment (MaksymR) [inline pom.xml:10]: the test comment: what about "<packaging>jar</packaging> ?)

**Review by MaksymR — CHANGES_REQUESTED** [review]
the test comment: what about "<packaging>jar</packaging> ?)

File: pom.xml (hunk 1 above)
Comment (MrScors) [reply pom.xml]: creat <finalName>FlightBoocking</finalName> in <build/> for docker builds

**Review by OlexiiNazarenko — DISMISSED** [review]

**Review by melkiy92 — APPROVED** [review]

**Review by MrScors — APPROVED** [review]
All fine

**Review by Oleksandr-Marchenko — APPROVED** [review]

**Review by kanivets39 — APPROVED** [review]

**Review by MaksymR — APPROVED** [review]

**Review by OlexiiNazarenko — APPROVED** [review]

**Pull request merged**